sources/settings-migrations/v1.39.0/kubelet-container-log-extra/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings for configuring the NVIDIA k8s device plugin.
+fn run() -> Result<()> {
+    migrate(AddSettingsMigration(&[
+        "settings.kubernetes.container-log-max-workers",
+        "settings.kubernetes.container-log-monitor-interval"
+    ]))
```

Comment:
Don't forget to add:

```suggestion
    migrate(AddSettingsMigration(&[
        "settings.kubernetes.container-log-max-workers",
        "settings.kubernetes.container-log-monitor-interval",
        "settings.kubernetes.single-process-oom-kill"
    ]))
```